core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -74,15 +75,17 @@ oppia.controller('StatisticsTab', [
     $scope.refreshExplorationStatistics = function(version) {
       if ($scope.ENABLE_NEW_STATS_FRAMEWORK) {
         $scope.explorationStatisticsUrl = (
-          '/createhandler/statistics/' + explorationData.explorationId);
+          '/createhandler/statistics/' + ExplorationDataService.explorationId);
       } else {
         $scope.explorationStatisticsUrl = (
-          '/createhandler/statistics_old/' + explorationData.explorationId +
+          '/createhandler/statistics_old/' + 
+          ExplorationDataService.explorationId +
           '/' + version);
```

Comment:
This can be moved to the previous line.

Reply:
Done

Reply:
@bching, why am I not able to see this change ? Are you sure you committed it ?

Reply:
@shubha1593 I'm sorry I rolled back some changes because of the Travis errors. I moved it to the previous line now.

Reply:
@shubha1593 I restarted the linter job on Travis but it appears it failed on the new commit. Perhaps it is a bit too long for the linter to pass?

Reply:
@bching Oops sorry, my bad! You're right.

Reply:
Oh no wait, you moved `ExplorationDataService.explorationId +` to previous line. I meant move `'/' + version);` to previous line like 
![update](https://user-images.githubusercontent.com/7807557/32695490-dae1a5d2-c783-11e7-9fbb-b85609134d3d.jpeg)

Reply:
@shubha1593 Ahhh, all right I have done so now.